src/runtimes/node/bundlers/none/index.ts

```diff
@@ -0,0 +1,109 @@
+import { dirname, extname, normalize } from 'path'
+
+import { FunctionBundlingUserError } from '../../../../utils/error.js'
+import { RuntimeType } from '../../../runtime.js'
+import { getBasePath } from '../../utils/base_path.js'
+import { filterExcludedPaths, getPathsOfIncludedFiles } from '../../utils/included_files.js'
+import { ModuleFormat } from '../../utils/module_format.js'
+import { getNodeSupportMatrix } from '../../utils/node_version.js'
+import { getPackageJsonIfAvailable } from '../../utils/package_json.js'
+import { BundleFunction, GetSrcFilesFunction, NodeBundlerType } from '../types.js'
+
+/**
+ * This bundler is a simple no-op bundler, that does no bundling at all.
+ * It returns the detected moduleFormat and the mainFile + includedFiles from the config.
+ */
+
+/**
+ * Mimics the logic from Node.js, as functions with this bundler will be executed as is in Node.js.
+ */
```

Comment:
I think I understand what you mean with this comment, but I find the formulation a bit confusing. Here's a suggestion with a link to the Node.js docs, but feel free to discard if you don't think it helps.

```suggestion
// Uses the same algorithm as the ESM_FILE_FORMAT resolver from Node.js to get
// the right module format for a function based on its main file.
// See https://nodejs.org/api/esm.html#resolver-algorithm-specification.
```